{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/comments/330347664", "html_url": "https://github.com/tensorflow/tensorflow/pull/11385#issuecomment-330347664", "issue_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/11385", "id": 330347664, "node_id": "MDEyOklzc3VlQ29tbWVudDMzMDM0NzY2NA==", "user": {"login": "drpngx", "id": 20959853, "node_id": "MDQ6VXNlcjIwOTU5ODUz", "avatar_url": "https://avatars1.githubusercontent.com/u/20959853?v=4", "gravatar_id": "", "url": "https://api.github.com/users/drpngx", "html_url": "https://github.com/drpngx", "followers_url": "https://api.github.com/users/drpngx/followers", "following_url": "https://api.github.com/users/drpngx/following{/other_user}", "gists_url": "https://api.github.com/users/drpngx/gists{/gist_id}", "starred_url": "https://api.github.com/users/drpngx/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/drpngx/subscriptions", "organizations_url": "https://api.github.com/users/drpngx/orgs", "repos_url": "https://api.github.com/users/drpngx/repos", "events_url": "https://api.github.com/users/drpngx/events{/privacy}", "received_events_url": "https://api.github.com/users/drpngx/received_events", "type": "User", "site_admin": false}, "created_at": "2017-09-18T20:33:34Z", "updated_at": "2017-09-18T20:33:34Z", "author_association": "MEMBER", "body_html": "<p>Yes, it's a transient failure that we get now and then. Since we want to test against windows still, I'll try again.</p>\n<p>Jenkins, test this please.</p>", "body_text": "Yes, it's a transient failure that we get now and then. Since we want to test against windows still, I'll try again.\nJenkins, test this please.", "body": "Yes, it's a transient failure that we get now and then. Since we want to test against windows still, I'll try again.\r\n\r\nJenkins, test this please."}